### PR TITLE
Add Prometheus monitoring tool

### PR DESCRIPTION
**Pull request by carlwolff**
Multi-purpose monitoring tool that fits perfectly in the list of monitoring software.

### CODE REVIEW

**Comment by Kickball** [comment]
Mind adding the primary programming language, then I can merge the pull request.

File: README.md [hunk 1]
```diff
@@ -321,6 +321,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [LibreNMS](http://www.librenms.org/) - Community-based GPL-licensed network monitoring system - `GPLv3`
   * [Nagios](https://www.nagios.org/) - Industry standard monitoring software, which has a lot of plugins and customisation posibilities. - `GPL`
   * [phpSysInfo](https://phpsysinfo.github.io/phpsysinfo/) - a customizable PHP script that displays information about your system nicely ([Demo](http://phpsysinfo.sourceforge.net/phpsysinfo/index.php?disp=bootstrap))
+  * [Prometheus](http://prometheus.io/) - Monitoring system and time series database. ([Source code](https://github.com/prometheus/prometheus)) - `Apache`
```
Comment (nodiscc) [inline README.md]: Can you add `Go` at line end?

**Comment by carlwolff** [comment]
@Kickball @nodiscc Of course, I'll update!

**Comment by carlwolff** [comment]
@Kickball Programming language added at the end.

**Comment by Kickball** [comment]
Thanks for the contribution!

**Comment by carlwolff** [comment]
...and thank you for maintaining a list like this. There's a bunch of great tools out there never otherwise seen. I guess the hardest part now is to keep bloat out in order to keep a curated list.

**Pull request merged**